scripts/in_container/run_tmux_welcome.sh

```diff
@@ -19,5 +19,7 @@ cd /opt/airflow/ || exit
 clear
 echo "Welcome to your tmux based running Airflow environment (courtesy of Breeze)."
 echo
-echo "     To stop Airflow and exit tmux just type 'stop_airflow'."
+echo "     To stop Airflow and exit tmux, just type 'stop_airflow'."
+echo
+echo "     If you want to rebuild webserver assets dynamically, run 'cd airflow/www; yarn dev'"
```

Comment:
Let's make sure they also install dependencies at this step: `yarn && yarn dev`

Reply:
I think they also need to use the `-d` flag with `airflow webserver`

Reply:
True. Done. it would be installed anyway (it also runs automatically the first time when you enter start-airflow now), but I agree it would not hurt to run yarn again here.

Reply:
Also added the `-d` flag info.